package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghost-storage-base",
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Could you pls revert the version bump? Thanks :)

Reply:
@kirrg001 Done.